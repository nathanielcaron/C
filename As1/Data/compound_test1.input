u 5
u 10
u 15
u 20
u 25
u 30
o
o
u 1
u 2
u 3
e
o
e
x
