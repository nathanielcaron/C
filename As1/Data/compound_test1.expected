5
10
15
20
25
failed push
25
20
1
2
failed push
2
2
1
Successfully executed 12 instructions
|_ 5 _|
|_ 10 _|
|_ 15 _|
|_ 1 _|
4 elements
